docs/Tutorials/Private-Network/Create-Private-Network.md

```diff
@@ -58,7 +58,7 @@ the `Private-Network` directory:
 ```json
 {
   "config": {
-      "constantinoplefixblock": 0,
+    "londonBlock": 0,
```

Comment:
if you're adjusting the indent here, the following line (ethash) should also move

Reply:
addressed